CRM/Export/BAO/ExportProcessor.php

```diff
@@ -2167,4 +2167,42 @@ protected function setGreetingStringsForSameAddressMerge($formValues) {
     }
   }
 
+  /**
+   * Create the temporary table for output.
+   *
+   * @return string
```

Comment:
Style checker says this `@return` statement is incorrect as the function does not return anything.